javascript-loops-practice/get-student-names.js

```diff
@@ -0,0 +1,9 @@
+/* exported getStudentNames */
+function getStudentNames(students) {
+  const arr = [];
+
+  for (const x in students) {
+    arr.push(students[x]);
```

Comment:
This doesn't quite work, It will return the entire object and you get back the same values in the array that you passed in.

You should only get the string value of each name as the index of each array .